### ISSUE TITLE
Неперекладене повідомлення

### ISSUE THREAD

**Issue by Maximaximum** [Virake/questforglory1_ukrainian#5]
https://ibb.co/eXbrHJ

"You find X silver coins, and carefully place them in your pouch"

**Comment by Vyhrovyk** [comment]
@Maximaximum, це перекладено і закодовано до гри, питання в іншому ви раптом не старе збереження використовували — до встановлення українізатора

**Comment by Maximaximum** [comment]
Ні, я з нуля купив гру на Steam, встановив її, і одразу ж встановив українізатор. І гру з нуля починав
